projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -116,38 +134,49 @@ private function add_connection_banner_actions() {
 
 	}
 
-	// Render the page with a common top and bottom part, and page specific content
-	function render() {
+	/**
+	 * Render the page with a common top and bottom part, and page specific content.
+	 */
+	public function render() {
 		// We're in an IDC: we need a decision made before we show the UI again.
 		if ( self::$block_page_rendering_for_idc ) {
 			return;
 		}
 
-		// Check if we are looking at the main dashboard
-		if ( isset( $_GET['page'] ) && 'jetpack' === $_GET['page'] ) {
+		// Check if we are looking at the main dashboard.
+		if ( isset( $_GET['page'] ) && 'jetpack' === $_GET['page'] ) {// phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
We normally have a space before the `//`, and it's good practice to comment why we're ignoring the nonce sniff.
```suggestion
		if ( isset( $_GET['page'] ) && 'jetpack' === $_GET['page'] ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended -- View logic.
```